model/src/main/kotlin/utils/SimplePackageConfigurationProvider.kt

```diff
@@ -48,9 +49,9 @@ class SimplePackageConfigurationProvider(
          * configuration per [Identifier] and [Provenance].
          */
         fun forDirectory(directory: File): SimplePackageConfigurationProvider {
-            val entries = findPackageConfigurationFiles(directory).mapTo(mutableListOf()) { file ->
+            val entries = findPackageConfigurationFiles(directory).mapNotNullTo(mutableListOf()) { file ->
```

Comment:
Before this change this function ensured that each file is a valid package configuration.
Now it doesn't anymore, but silently swallows empty file issues.

As we're using this function for our internal PSV to ensure that all files are package configuration files in the directory, I'd prefer to keep the previous semantics.

Reply:
> Before this change this function ensured that each file is a valid package configuration.

That's exactly the arguable point: Are empty configuration files invalid configuration files? Or are they valid configuration files containing no configuration? I tend to think the latter.

> I'd prefer to keep the previous semantics.

I'd be fine to leave `SimplePackageConfigurationProvider.kt` specially untouched, as our problem is mainly with empty `.ort.yml` files.

Reply:
> That's exactly the arguable point: Are empty configuration files invalid configuration files? Or are they valid configuration files containing no configuration? I tend to think the latter.

I believe that this can be decided case-by-case per file type. For package configurations I believe empty files do not make sense, because there must be the "matcher" bit specifying to which package this (potentially otherwise empty) configuration belongs to.